service/src/test/java/bio/terra/catalog/api/DatasetApiControllerTest.java

```diff
@@ -188,4 +191,14 @@ void getDatasetPreviewTable() throws Exception {
         .andExpect(content().contentType(MediaType.APPLICATION_JSON))
         .andExpect(jsonPath("$.columns[0].name").value(columnName));
   }
+
+  @Test
+  void exportMetadata() throws Exception {
+    var datasetId = new DatasetId(UUID.randomUUID());
+    var workspaceId = UUID.randomUUID();
+    doNothing().when(datasetService).exportDataset(user, datasetId, String.valueOf(workspaceId));
```

Comment:
I assume this has the same effect as doing a `verify()` after the `perform()`. Is there a reason to prefer one form over the other? It looks like other tests in this class use `verify()` to do this.